corehq/form_processor/backends/sql/processor.py

```diff
@@ -33,6 +33,7 @@ def store_attachments(cls, xform, attachments):
                 attachment_id=uuid.uuid4(),
                 content_type=attachment.content_type,
             )
+            xform_attachment.blob_bucket = xform_attachment.blobdb_bucket()
```

Comment:
> I don't know how postgres stores text fields...

If it's a "char" field then it's pre-allocated/fixed-length; a "varchar" requires 1 byte + length of string.